docs/concepts/thread_safety.md

```diff
@@ -76,13 +76,13 @@ The RTOS provides several mechanisms to move interrupt processing onto a thread.
 
 **Warning:** In mbed OS 5, if you attempt to use a mutex from within an interrupt nothing happens; attempts to lock a mutex will succeed immediately, regardless of whether the lock is actually free. In other words, if you acquire a mutex lock in an interrupt, you can break the thread safety mechanisms and introduce race-conditions into an otherwise safe piece of code. Future versions of mbed OS will provide warnings and ultimately prevent this from happening.
 
-For more information see [rtos/rtos/Mutex.h](https://github.com/mbedmicro/mbed/blob/master/rtos/rtos/Mutex.h).
+For more information see [rtos/Mutex.h](https://github.com/ARMmbed/mbed-os/blob/master/rtos/Mutex.h).
```

Comment:
Check if the API doxygen has all of these: https://docs.mbed.com/docs/mbed-os-api/en/mbed-os-5.3/api/index.html